Business/ValidationRules/FluentValidation/ProductValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FluentValidation;
+using WebNetSample.Entity.Concrete;
+
+namespace WebNetSample.Business.ValidationRules.FluentValidation;
+
+public class ProductValidator : AbstractValidator<Product>
+{
+    public ProductValidator()
+    {
+        RuleFor(product => product.Name)
+            .MinimumLength(3)
+            .MaximumLength(30)
+            .WithMessage("Product name must contain between 3-50 characters.");
```

Comment:
Looks like your message and validation rule have different values. To avoid such problems, try to put such values in constants